docpages/06_advanced_reference.md

```diff
@@ -4,6 +4,7 @@
 * \subpage thread-model
 * \subpage voice-model
 * \subpage coding-standards
+* \subpage docs_standards
```

Comment:
I think it should be `docs-standards` to mathc the style

Reply:
No, read the docs standard lol. We should we using snake_case as that's something we have in our code prs but we've never actually followed with docs.

Reply:
Brain has stated he'd like to use kebab-case for stuff like this, so we're now using `docs-standards`